File: php/Product.php
<?php

class Product
{
    private $idProduct = "";
    private $price = 0;

    public function __construct($idP="", $p=0)
    {
        $this->idProduct = $idP;
        $this->price = $p;
    }
    
    public function setIdProduct($idP = "")
    {
        $this->idProduct = $idP;
    }

    public function getIdProduct()
    {
        return $this->idProduct;
    }

    public function setPrice($p = 0)
    {
        $this->price = $p;
    }

    public function getPrice()
    {
        return $this->price;
    }
}

?>

File: python/Product.py
class Product():

    #private atribute
    __idProduct = ""
    __price = 0

    #constructor
    def __init__(self):
        self.__idProduct = ""
        self.__price = 0

    #membuat set dan get
    def setIdProduct(self, idP):
        self.__idProduct = idP

    def getIdProduct(self):
        return self.__idProduct
    
    def setPrice(self, p):
        self.__price = p
    
    def getPrice(self):
        return self.__price

    # Menampilkan atrubut kelas
    def showProduct(self):
        print("ID Product    : " + str(self.__idProduct))
        print("Price         : " + str(self.__price))